codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2Connection.java

```diff
@@ -959,4 +940,139 @@ private boolean isLocal() {
             return this == localEndpoint;
         }
     }
+
+    /**
+     * Provides an interface to manage the active streams for this {@link Http2Connection}.
+     */
+    private interface ActiveStreams {
```

Comment:
It allows the `Event` interface to nest inside of `ActiveStreams` where we couldn't put this nested `Event` interface inside the `DefaultActiveStreams`.  I can rename the `Event` interface to `ActiveStreamsEvent` and put it inside the `DefaultHttp2Connection` scope if that is preferred?